src/treemodel/position.js

```diff
@@ -411,6 +411,8 @@ export default class Position {
 	/**
 	 * Creates a new position after given node.
 	 *
+	 * @see core.treeModel.TreeWalkerValue
```

Comment:
Are you sure this is the correct way to use @see tag? I am not sure either and have seen various way to use it.